.travis.yml

```diff
@@ -1,13 +1,19 @@
 language: perl
 sudo: false
 perl:
+  - "5.26"
+  - "5.24"
+  - "5.22"
   - "5.20"
+  - "5.18"
   - "5.16"
+  - "5.14"
+  - "5.12"
```

Comment:
Why do you need so many perl versions?

Reply:
Was reflecting the same list from Mojo::Pg.

Reply:
Seems reasonable to include all the standard ones that are supported IMO

Reply:
I see. Thank you.